src/xpcc/driver/position/vl53l0.cpp

```diff
@@ -0,0 +1,106 @@
+// coding: utf-8
+/* Copyright (c) 2017, Christopher Durand, Niklas Hauser
+ * All Rights Reserved.
+ *
+ * The file is part of the xpcc library and is released under the 3-clause BSD
+ * license. See the file `LICENSE` for the full license governing this code.
+ */
+// ----------------------------------------------------------------------------
+
+#include "vl53l0.hpp"
+
+namespace xpcc
+{
+
+namespace vl53l0_private
+{
+
+FLASH_STORAGE(BinaryConfiguration configurationFlash[]) =
+{
+	// the infamous binary blob of black magic
```

Comment:
This line can also be found in the Vl6180 driver (vl6180.cpp). It might be appropriate to change it there.